packages/seacas/applications/exo_format/cmake/Dependencies.cmake

```diff
@@ -1,9 +1,3 @@
-if(CMAKE_PROJECT_NAME STREQUAL "SEACAS")
-  TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
-    LIB_REQUIRED_PACKAGES Exodus
-  )
```

Comment:
This should not be impacting the Trilinos project.  Not sure why this had to be pulled out.

Reply:
Found while looking for actual issue.  Did not need the if.  Dependency is always there.